template/README.md

```diff
@@ -59,6 +59,12 @@ If no additional dependencies is required, **this section can be removed**.
 $ pip install scipy torch
 ```
 
+If your package has `requirements.txt`, it will be uploaded to the OptunaHub at the publication time, and the package dependencies will be available to install as follows.
```

Comment:
The current documentation does not explain the publication process in detail, so contributors might not be familiar with "publication time" . Can we remove it?

```suggestion
If your package has `requirements.txt`, it will be automatically uploaded to the OptunaHub, and the package dependencies will be available to install as follows.
```

Reply:
I also added `automatically` , but it is not a strong opinion.

Reply:
Thank you for your feedback. I applied your suggestion.